EXT_BeamPhysics.md

```diff
@@ -0,0 +1,269 @@
+  `SpeciesType` extension.
+  - Example: `electron`, `H2O`.
+
+- `charge`
+  - Type: Optional *(int)*
+  - Description: The charge state of the particles. Not needed if the charge can be computed
+  from knowledge of the `SpeciesType`.
+
+- `latticeElementName`
+  - Type: Optional *(string)*
+  - Description: The name of the lattice element the particle or particles are in. This only makes sense if all
+  particles are in the same lattice element. Also see: `latticeElementID` and `locaitonInElement`.
+
+- `latticeElementID`
+  - Type Optional *(string)*
+  - Description: The ID string for the lattice element given by `latticeElementName`. The idea is that while more than
+    one lattice element may have the same name, the ID string will be unique.
+  - This, along with `locationInElement` sets the origin for specifying particle positions in the **lattice** coordinate system.
+  - Example: With [Bmad](https://www.classe.cornell.edu/bmad/) based programs the ID string is of the form
+    **branch-index>>element-index** where **branch-index** is the associated
+branch index integer, and **element-index** is the associated lattice element index within the branch.
+
+- `locationInElement`
+  - Type Optional *(int)*
+  - Description: This attribute is used with  `latticeElementID`/`latticeElementName` to specify
+  the origin where the particle or particles are measured with respect in the **lattice** coordinate system.   
+  - The origin is always on the longitudinal axis (x = y = 0) of the **lattice** coordinate system.
+  - Possible values:    
+    - `Upstream-End`: Upstream end of element outside of any edge fields.
+    - `<s-position>`: Where `<s-position>` is a number. Inside the element at a distance, given by `<s-position>`,
+    from the upsteam end of the element.
+    - `Downstream-End`: Downstream end of the element outside of any edge fields.
+  - Note: Since some programs will model edge fields of a lattice element as having zero length, the longitudinal **s**-position
+does not necessarily provide enough information to determine where a particle is with respect to an edge field so `Upstream-End` and `Downstream-End` are provided.
+
+- `momentumNormalization`
+    - Type: Optional *(string)*
+    - Description: Normalization used for momentum values.
+    - Possible values:
+        - `referenceTotalMomentum`: Normalize with respect to the `referenceTotalMomentum` attribute
+        - `referenceEnergy`: Normalize with respect to the `referenceEenergy` attribute.
+        - `none`: No normalization.
+
+- `referenceEnergy`
+    - Type: Optional *(float)* attribute.
+    - Description: Specifies the reference energy from which the `energy` is measured with respect to.
+
+- `referenceTotalMomentum`
+    - Type: Optional *(float)* attribute
+    - Description: Specifies the reference total momentum from which the total momentum is measured with respect to.
```

Comment:
Should we impose the unit of this attribute?

Reply:
Actually this should be moved to the `Non Per-Particle Records of the Particle Root Group` and become a record.

Reply:
Fixed in new draft to be uploaded soon...